### PR TITLE
Cleanup docs - fix formatting, typos

### PR DESCRIPTION
**Pull request by allewun**

### CODE REVIEW

**Comment by tirodkar** [comment]
Hey @allewun, could you please squash all your commits into one?

**Comment by allewun** [comment]
@tirodkar yep, done.

File: docs/install-and-run.md [hunk 1]
```diff
@@ -262,7 +261,7 @@ steps in your setup:
 
 1. Import EarlGrey in your test bundle's bridging header:
 
-   ```
+   ```objc
     #import <EarlGrey/EarlGrey.h>
```
Comment (tirodkar) [inline docs/install-and-run.md]: `#import <EarlGrey/EarlGrey.h>;`

**Comment by tirodkar** [comment]
@allewun Thanks for this!

**Pull request merged**